src/main.c

```diff
@@ -4206,6 +4275,49 @@ bool se_selectable_with_box(const char * first_label, const char* second_label,
 #endif
   return clicked; 
 }
+
+void se_boxed_image_dual_label(const char * first_label, const char* second_label, const char* box, sg_image image, int reduce_width, ImVec2 uv0, ImVec2 uv1){
+  ImVec2 win_min,win_sz,win_max;
+  win_min.x=0;
+  win_min.y=0;                                  // content boundaries min (roughly (0,0)-Scroll), in window coordinates
+  igGetWindowSize(&win_sz);
+  win_min.y+=igGetScrollY();
+  win_max.x = win_min.x+win_sz.x; 
+  win_max.y = win_min.y+win_sz.y;
+
+  int item_height = 50; 
+  int padding = 4; 
+
+  float disp_y_min = igGetCursorPosY();
+  float disp_y_max = disp_y_min+item_height+padding*2;
+  //Early out if not visible (helps for long lists)
+  if(disp_y_max<win_min.y||disp_y_min>win_max.y){
+    igSetCursorPosY(disp_y_max);
+    return;
+  }
+  int box_h = item_height-padding*2;
+  int box_w = box_h;
+  igPushIDStr(second_label);
+  ImVec2 curr_pos; 
+  igGetCursorPos(&curr_pos);
+  ImVec2 next_pos=curr_pos;
+  next_pos.y+=box_h+padding*2;
+  curr_pos.y+=padding; 
+  igSetCursorPos(curr_pos);
+
+  igSetCursorPosX(curr_pos.x+box_w+padding);
+  igSetCursorPosY(curr_pos.y-padding);
+  se_text(first_label);
```

Comment:
se_text/se_text_disabled work like printf statements. So, the first argument is a format string. This makes this code insecure as the variable first_label could contain a character escape sequence.

Sanitize this by using `se_text("%s",first_label)` instead.

Reply:
These are not actually unsafe as first_label is not a user provided input (so it couldn't contain an unintended character escape sequence) but I'll sanitize them anyway to avoid compiler warnings